posts/templatetags/query_params.py

```diff
@@ -0,0 +1,12 @@
+from urllib.parse import urlencode
+
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def append_query_param(context, **kwargs):
```

Comment:
though about also adding `*args` handling to pair items and add to dict as well, not sure, if needed.

Reply:
It's ok